blog/_posts/2018-07-07-Team101DaysOfSummerInroduction.md

```diff
@@ -0,0 +1,78 @@
+
+Gauri is an experienced Rust programmer who did her internship with Mozilla under the prestigious Outreachy programme. We came to know about her from the RGSoC page itself, where she had mentioned that she wanted to coach a team for Servo. She is a happy and cheerful person indeed. We plan to trouble her with lots of doubts this summer 😜
+
+<b>The journey till now - </b>
+
+We got to know about RGSoC when seniors from our college Prachi and Ipshita got selected for it last year. After that fine day, we wanted to be a part of such a program which could help us contribute to such a large community under the mentorship of talented and experienced people.
+
+So we made sure that we started looking for organisations on time. By mid-January, even before the organisations were finalized, we started making our mind on which project to apply for. We started understanding and making small contributions to our first project. But after few days came the shortlist of projects…….A small set back for us, the project that we wanted to apply for didn’t get selected :( 
+
+![Sad](/img/blog/2018/2018-07-07-101Daysgif2.gif)
+<div class="image-credits">Source: Giphy.com</div>
+
+But nevermind, we began looking for some other projects. 
+We took one more week, installing new code bases and getting in touch with mentors and the community, figuring out which project was the perfect match for us. We finally narrowed down on Servo, Mozilla. It was something scary and challenging for us:
+
+It offered us Rust, a completely different programming language. We had worked with C++ and Python before, but Rust was different. It had features from more than 10 different languages. Primarily it was safe, fast and concurrent!  😎
+It had a really large code base with more than 2000 live issues. Also, we had never ever got the unique opportunity of working with browser engines. 
+
+
+So we decided to get our hands dirty. We implemented our very first API `DOM Doge` Api in Servo. We were able to do it only because of the blog post written by one of the past RGSoC scholar who gave insights into how a basic API can be built and how the workflow of the whole API should be. This task was something that we were able to complete within a day (But it was literally a lot of effort, solving the small errors that came up each time we tried to compile the code.)
+
+
+Ahh, we were so happy :P
+
+![HappyAgain](/img/blog/2018/2018-07-07-101Daysgif3.gif)
+<div class="image-credits">Source: Giphy.com</div>
+
+![MissionCompleted](/img/blog/2018/2018-07-07-101Daysgif4.gif)
+<div class="image-credits">Source: Giphy.com</div>
+
```

Comment:
You need to use 'br' tag if you want one line space between images and next paragraph. :)